R4ML/R/r4ml.frame.R

```diff
@@ -259,7 +259,8 @@ setMethod("r4ml.impute",
     constants <- list()
     constant_names <- list()
     
-    for(col in names(df_columns)){
+    num_cols <- df_columns[df_columns == "mean"]
```

Comment:
This is the named list so we are collecting all of them which are the "mean" the output is not just the one element